check_dependencies.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This script checks for the major dependencies that RMG-Py requires.
+"""
+
+import re
+import subprocess
+
+missing = False
+
+print 'Checking vital dependencies...\n'
```

Comment:
Perhaps add `\n` before `Checking...` so the output is separated from the other make build

Reply:
Done